instrumentation/opentelemetry-instrumentation-aiohttp-server/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "opentelemetry-instrumentation-aiohttp-server"
+dynamic = ["version"]
+description = "Aiohttp server instrumentation for OpenTelemetry"
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.7"
+authors = [
+  { name = "OpenTelemetry Authors", email = "cncf-opentelemetry-contributors@lists.cncf.io"}
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11"
+]
+dependencies = [
+    "opentelemetry-api ~= 1.12",
+    "opentelemetry-instrumentation == 0.39b0.dev",
+    "opentelemetry-semantic-conventions == 0.39b0.dev",
+    "opentelemetry-util-http == 0.39b0.dev",
```

Comment:
Are these `.dev` dependencies correct? Why not depend on `0.39b0`?

Reply:
@kennytrytek-wf I used the dependencies from the `aiohttp-client` instrumentation as a guide to it.